src/index.js

```diff
@@ -3,7 +3,8 @@ import Sinon from 'sinon';
 import ReactWrapper from './ReactWrapper';
 import ShallowWrapper from './ShallowWrapper';
 import { onPrototype } from './Utils';
-import { renderToStaticMarkup } from './react-compat';
+import reactCompat from './react-compat';
+const {renderToStaticMarkup} = reactCompat;
```

Comment:
ES6 imports indeed can't be destructured inline, per spec - babel 5 violates the spec in this regard. babel 6 violates it in a different way.

Since the ES6 loader spec doesn't yet exist, ES6 modules don't yet exist, so technically, nobody should be using import/export anywhere under any circumstances. Since a number of people have chosen to use the syntax anyways, a number of compromises have to be made depending on the situation. This is appropriate here.